frontend/e-buddy/src/components/admin-panel/menu.tsx

```
@@ -28,129 +21,79 @@ export function Menu({ isOpen }: MenuProps) {
+                            isActive
+                              ? "bg-accent border-l-2 text-white"
+                              : "hover:bg-accent2-light hover:text-white"
+                          )}
+                        >
+                          <Icon className="h-5 w-5 flex-shrink-0" />
+                          {isOpen !== false && (
+                            <>
+                              <span className="ml-3 flex-1 truncate">
+                                {label}
+                              </span>
+                            </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)